sdk/storage/azure-storage-blobs/src/rest_client.cpp

```diff
@@ -4738,7 +4976,7 @@ namespace Azure { namespace Storage { namespace Blobs {
       {
         request.SetHeader("x-ms-lease-id", options.LeaseId.Value());
       }
-      request.SetHeader("x-ms-version", "2020-10-02");
+      request.SetHeader("x-ms-version", "2021-04-10");
```

Comment:
It needs to change API version in many places, why not define a constant?

Reply:
That is a good point. I also ever considered that, many times. Unfortunately, codeGen is not maintained by our team now. So...